core/commands/partials/aws_terraform_configurator.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require_relative '../../models/return_codes'
+require_relative '../../services/machine_configurator'
+require_relative '../../services/terraform_service'
+require_relative '../../models/network_settings'
+require_relative 'aws_terraform_configuration_generator'
+require_relative '../destroy_command'
+
+# The configurator brings up the configuration for the Vagrant
+class AwsTerraformConfigurator
+  include ReturnCodes
+
+  def initialize(config, env, logger)
+    @config = config
+    @env = env
+    @ui = logger
+    @provider = config.provider
+    @machine_configurator = MachineConfigurator.new(@ui)
+    @attempts = @env.attempts&.to_i || 5
+    @recreate_nodes = @env.recreate
+    @aws_service = @env.aws_service
+    @network_settings = if File.exist?(config.network_settings_file)
+                          NetworkSettings.from_file(config.network_settings_file)
+                        else
+                          NetworkSettings.new
+                        end
+  end
+
+  # Brings up nodes
+  #
+  # @return [Number] execution status
+  def up
+    nodes = @config.node_names
+    up_results = nodes.map { |node| bring_up_and_configure(node) }
+    return ERROR_RESULT unless up_results.detect(&:!).nil?
```

Comment:
You should instead return Result.error() or Result.ok(). It will be correctly parsed by the command executor.